web/src/components/tasks/IntercomScript.tsx

```diff
@@ -26,6 +43,9 @@ export const IntercomScript = ({
         data-user-name={user?.name}
         data-user-email={user?.email}
         data-user-operating-phase={operatingPhaseId}
+        data-user-legal-structure={legalStructureId}
+        data-user-industry-id={industryId}
+        data-user-business-persona={businessPersona}
```

Comment:
I know you created this file for the last intercom story, and I didn't notice this then, but I see now that this file lives in `components/tasks` directory. Is there a reason for that? It seems misleading since this component is used on every single page of the application, not just on tasks. I'd recommend moving this file